### DISCUSSION TITLE
How to read two zip files at once

### DISCUSSION THREAD

**Discussion by daxinhub** [EmulatorJS/EmulatorJS#863] [General]
It seems that Retroarch needs to read Kof.zip and Neogeo. zip in order to run Kof (if using a desktop emulator, these two files need to be placed in the same folder); But using EJS_gameURL can only retrieve the zip of ROM

![image](https://github.com/user-attachments/assets/d2245510-52a9-4ca6-bbd4-123aa0a6baf8)

**Comment by ethanaobrien** [comment]
[EJS_gameParentUrl](https://emulatorjs.org/docs/options#ejs_gameparenturl). This option will extract if it detects a zipped file. If you need it compressed, zip the zip file

**Reply by daxinhub** [reply]
> [EJS_gameParentUrl](https://emulatorjs.org/docs/options#ejs_gameparenturl). This option will extract if it detects a zipped file. If you need it compressed, zip the zip file

thanks,Can't multiple keys be pressed together? Some games require three or more keys to operate together

**Reply by ethanaobrien** [reply]
Yes, as I said to basically “zip the zip”, just zip your 2 or 3 files together, and they will all be extracted

**Reply by daxinhub** [reply]
> Yes, as I said to basically “zip the zip”, just zip your 2 or 3 files together, and they will all be extracted

thank you again, but can't emulatrojs play games by pressing multiple buttons at the same time? Some games require pressing three or more buttons to release skills

**Reply by ethanaobrien** [reply]
Yes, EmulatorJS will input as many inputs as the browser is telling it to.

if you are experiencing limitations with how many you can press, this will be a limitation of the browser, operating system, input driver, or the keyboard.

**Reply by daxinhub** [reply]
> Yes, EmulatorJS will input as many inputs as the browser is telling it to.
> 
> if you are experiencing limitations with how many you can press, this will be a limitation of the browser, operating system, input driver, or the keyboard.

Haha, it may be a keyboard issue. Try other computers later to see

**Comment by daxinhub** [comment]
🤣Why do I always have a bunch of unknown problems? Is there anything to pay attention to when referencing. NET
I can hear the sound of the game, but I can't see the visuals
![image](https://github.com/user-attachments/assets/504a5af0-5636-4bf1-adfc-652454482440)

**Reply by ethanaobrien** [reply]
Your EmulatorJS cores are outdated, enable EJS_DEBUG_XX

**Reply by daxinhub** [reply]
> Your EmulatorJS cores are outdated, enable EJS_DEBUG_XX

No, the core is all up-to-date; It should be due to Access Control Allow Origin or other reasons. In. NET 8, it seems that references to EJS_ path data and EJS_ core cannot directly retrieve local data (the emulator. min. js will report an error); But it's normal for me to switch to another computer, I can't understand.
And then ,EJSDEBUG_XX i cannot find where to enable it

**Reply by daxinhub** [reply]
If use  EJS_pathtodata = "https://cdn.emulatorjs.org/stable/data/" is used,it can run.
It seems to be a problem of obtaining the local core
